### DISCUSSION TITLE
MAUI Blazor Hybrid Support

### DISCUSSION THREAD

**Discussion by Md-Rohol-Amin** [sqlitecloud/discussions#52] [Feature request] [answered]
How to use sqlitecloud in MAUI blazor hybrid app? Is there any **SDK for MAUI blazor hybrid** ?

**Comment by Gioee** [comment, marked as answer]
Hi @Md-Rohol-Amin, no there isn't. But you can always use our weblite endpoints https://docs.sqlitecloud.io/docs/weblite in each tech stack we don't officially support.

**Reply by Md-Rohol-Amin** [reply]
yes, already i have done by edge function. thanks to **@Gioee** for answer.
